tests/UNMOCKED_end2end_tests/run_all_unmocked_tests.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+
+# Get the parameters
+export TOKEN=$1
+echo "token: ${TOKEN}"
+
+echo "############################### Download the clothing_dataset_small"
+
+git clone https://github.com/alexeygrigorev/clothing-dataset-small clothing_dataset_small
```

Comment:
Crashes the script if tests are ran multiple times because the folder already exists

Reply:
Fixed it with a check if the dataset already exists.